elements/bulbs-video/components/revealed.js

```diff
@@ -8,11 +8,10 @@ import Comscore from '../plugins/comscore';
 
 import React, { PropTypes } from 'react';
 import invariant from 'invariant';
+import SETTINGS from 'bulbs-elements/settings';
 
-// FIXME: where should this be defined? Per-app?
-//  Or in some better sort of settings file here?
-global.BULBS_ELEMENTS_ONIONSTUDIOS_GA_ID = 'UA-223393-14';
-global.BULBS_ELEMENTS_COMSCORE_ID = '6036328';
+global.BULBS_ELEMENTS_ONIONSTUDIOS_GA_ID = SETTINGS.BULBS_ELEMENTS_ONIONSTUDIOS_GA_ID;
+global.BULBS_ELEMENTS_COMSCORE_ID = SETTINGS.BULBS_ELEMENTS_COMSCORE_ID;
```

Comment:
Rather than dump this on settings, I think we should just read it from `SETTINGS` every time we use these values.